400-claims.md

```diff
@@ -105,6 +109,8 @@ The fields above are defined as follows:
 - `custom` (OPTIONAL): A section for custom extension data applicable to a given runtime.
 - `id` (REQUIRED): The claim id. A [ULID](https://github.com/ulid/spec) that MUST change with each new claim, so that every claim associated with an installation has a unique id. This is used to associate the the claim with its result(s). 
 - `installation` (REQUIRED): The name of the _installation_. This can be automatically generated, though humans may need to interact with it. It MUST be unique within the installation environment, though that constraint MUST be imposed externally. Elsewhere, this field is referenced as the _installation name_. The format of this field must follow the same format used for the `installation` field in the [bundle.json file specification](101-bundle-json.md#the-bundlejson-file).
+- `labels` (OPTIONAL): Labels are a set of key/value pairs of type string that MAY be used for querying. Labels defined on an installation MUST be included in the claim.
```

Comment:
> Labels defined on an installation MUST be included in the claim.

This is a dirty hack because we do not have a document that represents the installation. Each tool _could_ make up their own representation but that isn't really any better. So I am having the claim persist the installation labels just like it does the installation name already.